.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -292,44 +288,53 @@ kernel_matrix_testing_setup_env:
     INFRA_ENV: "aws/agent-qa"
     PIPELINE_ID: $CI_PIPELINE_ID
     TEAM: "ebpf-platform"
+    RESOURCE_TAGS: "instance-type:${INSTANCE_TYPE},arch:${ARCH}"
```

Comment:
Can you confirm this works with curly braces? (it should according to their docs, but I swear I had problems with them)

Reply:
The resource tags are added as intended. For example:
![image](https://github.com/DataDog/datadog-agent/assets/29705745/dd50f453-157a-4e74-ae9c-38c81070c79a)